src/app/translate.tsx

```diff
@@ -0,0 +1,177 @@
+"use client";
+
+import React, { useEffect, useState, useCallback } from "react";
+import defaultTranslations from "@/i18n/translations.en.json";
+import { getNestedTranslation, TranslationObject } from "@/app/utils";
+
+const languages = {
+  en: "English",
+  es: "Español",
+  // ...
+};
```

Comment:
https://jsfiddle.net/f2qhd63w/ gets the languages to show in the select in English, but getting them translated would require a string like the one the Desktop app has, and the API token is my own Crowdin API token (with read permissions for the Ruffle project).